newdle/api.py

```diff
@@ -237,3 +237,8 @@ def create_anonymous_participant(args, code):
     newdle.participants.add(participant)
     db.session.commit()
     return ParticipantSchema().jsonify(participant)
+
+
+@api.route('/newdle/<code>/send-result-emails', methods=('POST',))
+def send_result_emails(code):
+    return jsonify({})
```

Comment:
`return '', 204` ("no content") is probably fine here for now.

At some point it could be nice to return more detailed information, e.g. if sending an email to someone failed, but for the prototype that's too much. Failing the request is a bit ugly since you won't really know that the newdle has actually been created, and most likely some people got emails, but it's a prototype..